src/actions/ModelCreateAction.php

```
@@ -0,0 +1,62 @@
+
+use hipanel\actions\SmartCreateAction;
+use hipanel\modules\stock\models\HardwareSettings;
+use Yii;
+
+class ModelCreateAction extends SmartCreateAction
+{
+    public function saveCollection()
+    {
+        $pendingHardwareSettings = $this->collectHardwareSettings();
+
+        $result = parent::saveCollection();
```

PHPStan (2.1.54)

[error] hipanel\modules\stock\actions\ModelCreateAction::saveCollection() calls parent::saveCollection() but hipanel\modules\stock\actions\ModelCreateAction does not extend any class.


(class.noParent)